xc7/make/arch_define.cmake

```diff
@@ -26,6 +26,8 @@ function(ADD_XC7_ARCH_DEFINE)
       --clustering_pin_feasibility_filter off
       --disable_check_route on
       --strict_checks off
+      --allow_dangling_combinational_nodes on
+      --disable_errors check_unbuffered_edges:check_route
```

Comment:
Can we also disable the warnings that generate ~250k warnings?

Reply:
Done, the only thing is that I have specified the `suppress_warnings` option in `device.cmake` as they could be suppressed also for ice40 I suppose.
Another reason is that I didn't manage to get cmake to set the noisy_warnings.log output file to the build directory destination.